### PR TITLE
Store claims

### PR DESCRIPTION
**Pull request by faissalMT**

### CODE REVIEW

File: lib/local_authority/gateway/sequel_claim.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
+class LocalAuthority::Gateway::SequelClaim
+  def initialize(database:)
+    @database = database
+  end
+
+  def create(claim)
+    @database[:claims].insert(
+      project_id: claim.project_id,
+      type: claim.type,
+      status: claim.status,
```
Comment (elcorbs) [inline lib/local_authority/gateway/sequel_claim.rb]: Should bid id not be here aswell?

Comment (faissalMT) [reply lib/local_authority/gateway/sequel_claim.rb]: Bid id comes from the projects table, we get it when we join with projects https://github.com/homes-england/monitor-api/blob/c3c3700d1359eb051c8d64389699bee31f101ad6/lib/local_authority/gateway/sequel_claim.rb#L20

File: spec/unit/ui/use_case/convert_ui_ac_return_spec.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 describe UI::UseCase::ConvertUIACReturn do
-  let(:return_to_convert) do
+  let(:claim_to_convert) do
```
Comment (elcorbs) [inline spec/unit/ui/use_case/convert_ui_ac_return_spec.rb]: This should still be return - or the file name/ class it's testing is wrong

File: lib/local_authority/gateway/sequel_claim.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+    LocalAuthority::Domain::Claim.new.tap do |claim|
+      claim.project_id = row[:project_id]
+      claim.type = row[:type]
+      claim.bid_id = row[:bid_id]
+      claim.status = row[:claims_status]
+      claim.data = Common::DeepSymbolizeKeys.to_symbolized_hash(row[:claims_data].to_h)
+    end
+  end
+
```
[the remark this answers is not in the document]
Comment (faissalMT) [reply lib/local_authority/gateway/sequel_claim.rb]: Yes.

**Pull request merged**